cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -210,13 +210,12 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
 
     }
     section(title="Wait options", id="administration-wait-nowait", role = "enterprise-edition") {
-      p("""Aside from `SHOW DATABASES`, the database management commands all accept an optional
-          |`WAIT`/`NOWAIT` clause. The `WAIT`/`NOWAIT` clause allows a user to specify whether to wait
-          |for the command to complete before returning. The options are:
+      p("""Aside from `SHOW DATABASES`, all database management commands accept an optional
```

Comment:
Actually this text is out-of-date. In 4.4 we added ALTER DATABASE which also does not have a WAIT clause. So while you are anyway at it, you might update this to 'Aside from `SHOW DATABASES` and `ALTER DATABASE`, all management ...' in 4.4 and dev.